packages/edit-site/src/components/page-templates/index.js

```diff
@@ -45,9 +57,15 @@ const defaultLayouts = {
 	},
 	[ LAYOUT_GRID ]: {
 		showMedia: true,
+		layout: {
+			badgeFields: [ 'active', 'slug' ],
```

Comment:
Instead of introducing `__returnBadge` we should allow passing props to the badge component via this config. For example:

```js
{
  layout: {
    badgeFields: [ { field: 'active', intent: 'something' }, 'slug' ] 
  }
}
```